tests/test_view.py

```diff
@@ -111,190 +103,231 @@ def normalize(self, vec):
         return vec / np.linalg.norm(vec)
 
 
+@unittest.skipIf(GITHUB_ACTIONS, "GUI is not available in GitHub Actions")
+class ViewCommonCameraTC(ViewCameraTB, unittest.TestCase):
+    camera_type = "fps"  # no difference when use orbit camera
+
+    def setUp(self):
+        self.camera.reset()
+
+    def test_value_get_set(self):
+        c = self.camera
+
+        c.linear_speed = 123.0
+        self.assertEqual(c.linear_speed, 123.0)
+
+        c.look_speed = 456.0
+        self.assertEqual(c.look_speed, 456.0)
+
+    def test_vector_get_set(self):
+        c = self.camera
+
+        c.position = (1, 2, 3)
+        c.view_center = (4, 5, 6)
+        c.up_vector = (7, 8, 9)
+
+        self.assertEqual(c.position, (1, 2, 3))
+        self.assertEqual(c.view_center, (4, 5, 6))
+        self.assertEqual(c.up_vector, (7, 8, 9))
+
+    def test_default_values(self):
+        c = self.camera
+
+        c.default_position = (1, 2, 3)
+        c.default_view_center = (4, 5, 6)
+        c.default_up_vector = (7, 8, 9)
+        c.default_linear_speed = 123.0
+        c.default_look_speed = 456.0
+
+        c.reset()
+
+        self.assertEqual(c.position, (1, 2, 3))
+        self.assertEqual(c.view_center, (4, 5, 6))
+        self.assertEqual(c.up_vector, (7, 8, 9))
+        self.assertEqual(c.linear_speed, 123.0)
+        self.assertEqual(c.look_speed, 456.0)
+
+
 @unittest.skipIf(GITHUB_ACTIONS, "GUI is not available in GitHub Actions")
 class ViewFPSCameraTC(ViewCameraTB, unittest.TestCase):
     camera_type = "fps"
 
     def setUp(self):
-        self.widget.resetCamera()
+        self.camera.reset()
 
     def test_reset(self):
-        dt = 0.01
-        initial_pos = self.pos()
-        initial_view_vector = self.view_vector()
-        initial_view_center = self.view_center()
-        initial_up_vector = self.up_vector()
+        c = self.camera
```

Comment:
refactored all camera tests:
* removed usage of `dt`
* use properties instead of getters and setters